concepts/slices/about.md

```diff
@@ -32,6 +32,12 @@ newSlice := withData[2:]  // newSlice == []int{2,3,4,5}
 newSlice := withData[:]   // newSlice == []int{0,1,2,3,4,5}
 ```
 
+You can merge two or more slices with the `append` function. For this, you need to pass next slices as argument of variadic function.
+```go
+nextSlice := [int]{100,101,102}
+newSlice  := append(withData, nextSlice...) // newSlice == []int{0,1,2,3,4,5,100,101,102}}
```

Comment:
```suggestion
append(withData, nextSlice...)
// => []int{0,1,2,3,4,5,100,101,102}}
```

Since we wrote this exercise, we have agreed on a new format to present these kind of results. You can read more about this [here](https://github.com/exercism/go/issues/2202). This suggestion makes this example follow the new format.

However, if we make this change here, I think it's best to make the other examples in this exercise be consistent with this new format. If you want, feel free to make this change of format in the other examples too in this concept/exercise. This change to the examples can be another PR if you prefer, in which case feel free to ignore this particular suggestion.